pegasus/sites.v3/code.org/public/ai/teaching-assistant.haml

```diff
@@ -13,24 +13,70 @@ theme: responsive_full_width
       %p.white
         =hoc_s("ai_ta_page.desc")
     %figure
-      %img{src: "/images/ai/page/teaching-assistant/ai-ta-banner.png", alt: "", style: "width: 100%; margin-block-start: -2rem; margin-inline-start: -2rem;"}
+      %img{src: "/images/ai/page/teaching-assistant/ai-ta-banner-winner.png", alt: "", style: "width: 100%; margin-block-start: -2rem; margin-inline-start: -2rem;"}
+
+%section.bg-neutral-light
+  .wrapper
+    .flex-container.justify-space-between.align-items-center.gap-2.wrap
+      %div.col-47
+        = view :display_video_thumbnail, id: "intro_ta", video_code: "Lj7uZx-BKhw", play_button: 'center', letterbox: 'false'
+      .text-wrapper.col-47
+        %h2=hoc_s("ai_ta_page.get_started.header")
+        %ol
+          %li=hoc_s("ai_ta_page.get_started.watch_demo")
+          %li=hoc_s("ai_ta_page.get_started.assign_csd_3")
+          %li
+            %a{href: "https://form.jotform.com/240863841362155"}
+              = hoc_s("ai_ta_page.get_started.become_verified")
+          %li=hoc_s("ai_ta_page.get_started.use_rubric")
+        .content-footer
+          %a.link-button{href: "https://studio.code.org/s/csd3-2024"}
+            = hoc_s("ai_ta_page.get_started.use_ta")
```

Comment:
Nit: move this into a view too. This makes it easier to update sections if/when the time comes.